testing.rst

```diff
@@ -567,6 +567,56 @@ command.
     If the information you need to check is available from the profiler, use
     it instead.
 
+Simulating Login
+~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The ``loginUser()`` method was introduced in Symfony 5.1.
+
+When you want to add functional tests for protected pages, you have to
+first "login" as a user. Reproducing the actual steps - such as
+submitting a login form - make a test very slow. For this reason, Symfony
+provides a ``loginUser()`` method to simulate logging in in your functional
+tests.
+
+First, you have to have a test user and retrieve this user in your test.
```

Comment:
This phrase reads a bit confusing. Proposal:

```
Instead of login in with real users, it's recommended to create a user only for tests.
You can do that with Doctrine :ref:`data fixtures <user-data-fixture>`, to load the
testing users only in the test database.
```